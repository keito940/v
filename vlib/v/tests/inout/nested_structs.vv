module main

struct B {
	pass bool
	name string
}

fn (b &B) print() {
	println(b)
}

struct A {
	test bool
	b B
}

fn main() {
	a := A{}
	println(a)
	b := B{}
	b.print()
}
