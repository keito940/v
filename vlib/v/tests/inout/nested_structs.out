A {
    test: false
    b: B {
        pass: false
        name: ''
    }
}
B {
    pass: false
    name: ''
}